server/routers/status-page-router.js

```diff
@@ -85,7 +85,7 @@ router.get("/api/status-page/heartbeat/:slug", cache("1 minutes"), async (reques
                     SELECT * FROM heartbeat
                     WHERE monitor_id = ?
                     ORDER BY time DESC
-                    LIMIT 50
+                    LIMIT 100
```

Comment:
I set `LIMIT 100` to ensure there are no display issues even when using a 4K display.

Here is a screenshot. It would be helpful if the reviewer could also verify the functionality.
<img width="1346" alt="image" src="https://github.com/user-attachments/assets/4650008c-365e-4bd5-bbdf-962b1ab6e74c">

The `maxBeat` for the heart beat is calculated as follows. I temporarily added the log output to confirm the values.
https://github.com/louislam/uptime-kuma/blob/79a26180af29890e9050100add2877872c821105/src/components/HeartbeatBar.vue#L259

temporarily log output:
```
console.info({
    maxBeat: this.maxBeat,
    clientWidth: this.$refs.wrap.clientWidth,
    beatWidth: this.beatWidth,
    beatHoverAreaPadding2x: this.beatHoverAreaPadding * 2,
});
```